src/components/base/Spoiler/index.js

```diff
@@ -41,15 +41,17 @@ class Spoiler extends PureComponent<Props, State> {
   toggle = () => this.setState({ isOpened: !this.state.isOpened })
 
   render() {
-    const { title, children } = this.props
+    const { title, children, ...p } = this.props
     const { isOpened } = this.state
     return (
       <Fragment>
-        <Box horizontal flow={1} color="dark" align="center">
+        <Box horizontal flow={1} color="dark" align="center" {...p}>
           <IconContainer isOpened={isOpened}>
             <IconChevronRight size={12} />
           </IconContainer>
-          <Title onClick={this.toggle}>{title}</Title>
+          <Title {...p} onClick={this.toggle}>
```

Comment:
this is not a so good practice to explode `...p`, especially if it goes into multiple parts, but that's ok for now